engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -65,7 +74,18 @@ public void resetCamera() {
     }
 
     @ReceiveEvent
-    public void resetCamera(ResetCameraEvent resetCameraEvent, EntityRef client, AutoMountCameraComponent autoMountCameraComponent, ClientComponent clientComponent) {
+    public void resetCamera(ResetCameraEvent resetCameraEvent, EntityRef entityRef, AutoMountCameraComponent autoMountCameraComponent, ClientComponent clientComponent) {
+        if (localPlayer.getClientEntity().equals(entityRef)) {
+            clientComponent.camera.destroy();
+            clientComponent.camera = EntityRef.NULL;
+            // this will trigger a ClientComponent change which will in turn trigger a remount
+            localPlayer.getClientEntity().saveComponent(clientComponent);
+        }
+    }
+
+    @Command(shortDescription = "Mounts the camera to the character", requiredPermission = PermissionManager.NO_PERMISSION)
```

Comment:
Correct, it is a debugging tool.  But not one that causes any bad effects as it will only touch client side entities.  I dont exactly know what cases one will need to re-mount the camera as opposed to resetting the camera.  Do you think I should avoid making it a command so that there is less command noise?